content/en/platform/corda/4.10/community/release-notes.md

```diff
@@ -17,55 +17,79 @@ tags:
- 
-* Some RPCs provided by the node are now “quick” in that they bypass the standard RPC thread pool and will return relatively quickly even if the node is busy servicing a backlog of RPC requests. The affected RPCs are currentNodeTime(), and fetching the protocol version.
+For more information about platform versions, see [Versioning](versioning.md).
 
-* Previously, if a node failed to open an AMQP connection to a peer node if there was a failure due to a problem with the TLS handshake, it was possible for the peer to be permanently blocked such that further connection attempts would not be attempted unless the node was restarted.
+## New features and enhancements
 
-  With this update, peer nodes are now not permanently blocked but connections are retried using longer intervals - 5x 5 minutes, and then once a day.
-  
-* Warning messages from Artemis are no longer written to the standard output when disconnecting an SSH client from the node. The warnings are still written to the node’s log file though.
+### New Service Lifecycle Event
 
-* Testing with the YourKit tool showed high memory usage when creating tokens in a CorDapp. High memory usage was also seen when a node is restarted and inMemory selection was activated in the node. A fix was implemented and the same tests afterwards showed a major decrease in memory usage.
+During startup the node publishes a new service lifecycle event BEFORE_STATE_MACHINE_START immediately prior to starting the state machine, and will not start the state machine until all recipients of the event have handled it.
 
-* A new node configuration option, *cryptoServiceFlowRetryCount*, has been introduced.
+### Quick RPC for node health check
 
-  Previously, flows that suffered any variant of CryptoServiceException were admitted to the flow hospital for processing. The flow was retried a maximum of two times, and if it still failed then the exception was propagated back to the code that invoked the flow and the flow failed. 
-  
-  Now, *cryptoServiceFlowRetryCount* can be used to override the above default actions. 
+Some RPCs provided by the node are now “quick” in that they bypass the standard RPC thread pool and will return relatively quickly even if the node is busy servicing a backlog of RPC requests. The affected RPCs are currentNodeTime(), and getProtocolVersion().
+
+### Peer nodes not permanently blocked
+
+Previously, if a node failed to open an AMQP connection to a peer node if there was a failure due to a problem with the TLS handshake, it was possible for the peer to be permanently blocked such that further connection attempts would not be attempted unless the node was restarted. With this update, peer nodes are now not permanently blocked but connections are retried using longer intervals - 5x 5 minutes, and then once a day.
```

Comment:
```suggestion
Previously, if a node failed to open an AMQP connection to a peer node because there was a failure due to a problem with the TLS handshake, it was possible for the peer to be permanently blocked such that further connection attempts would not be attempted unless the node was restarted. With this update, peer nodes are now not permanently blocked but connections are retried using longer intervals - 5x 5 minutes, and then once a day.
```

Reply:
Done.